python/calico/felix/endpoint.py

```diff
@@ -887,16 +893,26 @@ def _apply_endpoint_update(self):
 
             tiers = pending_endpoint.get("tiers", [])
             tier_dict = OrderedDict()
+            raw_tier_dict = OrderedDict()
             for tier in tiers or []:
                 pols = []
+                raw_pols = []
                 for pol in tier["policies"] or []:
                     pol_id = TieredPolicyId(tier["name"], pol)
-                    pols.append(pol_id)
+                    if is_untracked(pol_id):
+                        raw_pols.append(pol_id)
+                    else:
+                        pols.append(pol_id)
                 tier_dict[tier["name"]] = pols
+                raw_tier_dict[tier["name"]] = raw_pols
```

Comment:
This might not work as expected.  Felix's normal rule is that a packet skips a tier if none of the policies match the endpoint, and there's a default drop at the end of the tier.

I think this might create a tier with no policies in the raw table for each tier in the filter table.  I think that tier will then drop by default.